contracts/factory/EVMScriptRegistryFactory.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity 0.4.18;
+
+import "../evmscript/EVMScriptRegistry.sol";
+
+import "../evmscript/executors/CallsScript.sol";
+import "../evmscript/executors/DelegateScript.sol";
+import "../evmscript/executors/DeployDelegateScript.sol";
+
+import "./AppProxyFactory.sol";
+import "../kernel/Kernel.sol";
+import "../acl/ACL.sol";
+
+
+contract EVMScriptRegistryFactory is AppProxyFactory, EVMScriptRegistryConstants {
+    address public baseReg;
+    address public baseCalls;
+    address public baseDel;
+    address public baseDeployDel;
+
+    function EVMScriptRegistryFactory() public {
```

Comment:
In general, it might be nice for factories to take in the addresses of any base contracts they need to avoid recreating the same base contracts in multiple deployments. If `0x0` is given, they should deploy a new version.